Source/FacilityManagementWebsite/SmartHotel.FacilityManagementWeb/SmartHotel.FacilityManagementWeb/ClientApp/src/app/tsi-chart/tsi-chart.component.css

```diff
@@ -0,0 +1,11 @@
+/* @import '~tsiclient/tsiclient.css'; */
+
+.chart-rendering-content {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  height: 500px;
```

Comment:
Think we should try for this component to "fill" the space it's given and have whoever is creating an instance of this component specify its dimensions, **if possible that is**

Reply:
Tried but needs more work to accomplish that. TSI chart "likes" explicit height.